src/back/hlsl/writer.rs

```diff
@@ -1,16 +1,22 @@
 //TODO: temp
 #![allow(dead_code)]
+use super::Options;
 use super::{Error, ShaderModel};
 use crate::back::hlsl::keywords::RESERVED;
+use crate::back::vector_size_str;
```

Comment:
Ideally, the code style here would follow other backends such as MSL and SPIR-V specifically.
But at the very minimum, let's combine the use cases so that there is one per crate. So `use crate::` is there only once.

Reply:
`rust-analyzer` add one line per "Import item ...", and don't automatically merge it. I always forget to merge it manually.